### DISCUSSION TITLE
Issue with ImageColumn Configuration in Laravel 10 with Livewire 3 and Filament 3

### DISCUSSION THREAD

**Discussion by shimaMatz** [filamentphp/filament#10337] [General]
I'm creating an admin panel using Laravel 10 with Livewire 3 and Filament 3. I have a specific issue related to the configuration of the ImageColumn in my code.

Here's the code snippet I'm using:

```
use Filament\Tables\Columns\ImageColumn;
 
ImageColumn::make('colleagues.avatar')
    ->circular()
    ->stacked()

```

Despite using the ->circular()->stacked() configuration, the images are not overlapping as expected. I want to confirm whether the code I've provided is correct and should result in overlapping images.

Please let me know if there are any issues with the provided code and if there's anything I need to adjust to achieve the desired image overlap effect.

**Comment by ShaungBhone** [comment]
Yes, your code is correct. What's your error? The issue is not specific.

**Comment by shimaMatz** [comment]
This code defines a table with multiple ImageColumns, each configured to be circular and stacked. 
However, despite the images being displayed, they are not overlapping as intended. 
If there are any issues or adjustments needed to achieve the desired image overlap effect, please let me know.

```
public static function table(Table $table): Table
{
    return $table
        ->columns([
            ImageColumn::make('style_1.styles_square')
                ->circular()
                ->stacked(),
            ImageColumn::make('style_2.styles_square')
                ->circular()
                ->stacked(),
            ImageColumn::make('style_3.styles_square')
                ->circular()
                ->stacked(),
            ImageColumn::make('style_4.styles_square')
                ->circular()
                ->stacked(),
            ImageColumn::make('style_5.styles_square')
                ->circular()
                ->stacked(),
        ])
        ->filters([
            //
        ])
        ->actions([
            Tables\Actions\EditAction::make(),
        ])
        ->bulkActions([
            Tables\Actions\BulkActionGroup::make([
                Tables\Actions\DeleteBulkAction::make(),
            ]),
        ]);
}

```

**Reply by ShaungBhone** [reply]
"I think 'stacked' is not like that. Maybe the documentation mentions that your image is multiple, which might be an array."

**Reply by shimaMatz** [reply]
Thank you very much. Just as you said, it is an array. Your answer was very helpful.
